src-ui/src/app/guards/dirty-form.guard.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { DirtyCheckGuard } from '@ngneat/dirty-check-forms';
+import { Observable, Subject } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { ConfirmDialogComponent } from 'src/app/components/common/confirm-dialog/confirm-dialog.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592507)
```suggestion
import { ConfirmDialogComponent } from "src/app/components/common/confirm-dialog/confirm-dialog.component";
```